templates/zerver/find_my_team.html

```diff
@@ -31,7 +33,7 @@ <h3 class="find-team-page-header">{{ _("Find your team") }}…</h3>
         {{ csrf_input }}
         <input type="text" class="form-control required"
                placeholder="{{ _("Enter email addresses") }}"
-               id="emails" name="emails"/>
+               autofocus="autofocus" id="emails" name="emails"/>
```

Comment:
You just need `autofocus`, not `autofocus="autofocus"` here (you can find this out by just googling for "html autofocus").